community/aqua/chemistry/QSE_pytket.ipynb

```diff
@@ -0,0 +1,381 @@
+    "\n",
+    "![alt text](LiHCompilation.png \"Title\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_aqua import QuantumInstance\n",
+    "from qiskit.transpiler import PassManager\n",
+    "from pytket.qiskit import TketPass\n",
+    "\n",
+    "pass_manager = PassManager()\n",
+    "tk_pass = TketPass(backend)\n",
+    "pass_manager.add_passes(tk_pass)\n",
+    "\n",
+    "quantum_instance = QuantumInstance(backend, pass_manager=pass_manager)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Perform classical calculation <a id='sectionB'></a>\n",
+    "\n",
+    "We execute our classical chemistry driver to obtain the hamiltonian integrals. In this case, we choose PYSCF as our driver, so make sure you have that installed.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Molecular repulsion energy:  0.7559674441714287\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_aqua_chemistry.drivers import ConfigurationManager\n",
+    "\n",
+    "# using driver to get fermionic Hamiltonian\n",
+    "# PySCF example\n",
+    "cfg_mgr = ConfigurationManager()\n",
+    "driver = cfg_mgr.get_driver_instance('PYSCF')\n",
+    "\n",
+    "pyscf_cfg = {'atom': base_molecule_str.format(bond_length),\n",
+    "             'unit': 'Angstrom',\n",
+    "             'charge': charge,\n",
+    "             'spin': spin,\n",
+    "             'basis': basis}\n",
+    "\n",
+    "molecule = driver.run({'properties': pyscf_cfg})\n",
+    "\n",
+    "print(\"Molecular repulsion energy: \", molecule._nuclear_repulsion_energy)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set up initial state and variational form <a id='sectionB'></a>\n",
+    "\n",
+    "We define the molecular hamiltonian and map it to a qubit operator according to some appropriate scheme such as the Jordan-Wigner transform. We initialize our initial state (a Hartree-Fock state) and our variational ansatz (a unitary coupled cluster with single and double excitations, UCCSD)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_aqua_chemistry import FermionicOperator\n",
+    "from qiskit_aqua_chemistry.aqua_extensions.components.initial_states import HartreeFock\n",
+    "from qiskit_aqua_chemistry.aqua_extensions.components.variational_forms import UCCSD\n",
+    "\n",
+    "n_qubits = molecule._one_body_integrals.shape[0]\n",
+    "n_electrons = molecule._num_alpha + molecule._num_beta - molecule._molecular_charge\n",
```

Comment:
do you mind updating the way accessing the property of molecule info? (the newest Aqua-chemistry had updated that.)
now, you need to access it without the underscore. i.e. `molecule.num_alpha`, and all other properties.